privacyIDEAADFSProvider/AdapterPresentationForm.cs

```diff
@@ -14,6 +14,29 @@ class AdapterPresentationForm : IAdapterPresentationForm
         public string WebAuthnSignRequest { get; set; } = "";
         public string OtpAvailable { get; set; } = "1";
         public string AuthCounter { get; set; } = "0";
+        public bool EnableEnrollment { get; set; } = false;
```

Comment:
i dont think this variable is required. The information can be derived from EnrollmentUrl and EnrollmentImg